zmq.go

```diff
@@ -15,10 +15,49 @@ import (
 	"time"
 )
 
-func writeAll(w io.Writer, buf []byte) error {
+var (
+	// MaxBodySize is the maximum frame size we're willing to accept.
+	// The maximum size of a Bitcoin block with all witness information is
+	// 4,000,000 bytes.
+	MaxBodySize uint64 = 4000000
```

Comment:
Perhaps we should increase this to a higher value: to ensure we don't get tripped up on any signalling overhead, and also to make the library more generic.

Reply:
Fixed to 32MiB.